test/OpenTelemetry.ClrProfiler.Managed.Loader.Tests/StartupTests.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.IO;
+using Xunit;
+
+namespace OpenTelemetry.ClrProfiler.Managed.Loader.Tests
+{
+    public class StartupTests
+    {
+        [Fact]
+        public void Ctor_LoadsManagedAssembly()
+        {
+            var director = Directory.GetCurrentDirectory();
+            Environment.SetEnvironmentVariable("OTEL_DOTNET_TRACER_HOME", @$"{director}\Profiler");
+
+            var exception = Record.Exception(() => Startup.ManagedProfilerDirectory);
+            // That means the assembly was loaded successfully and Initialize method was called.
+            Assert.Null(exception);
```

Comment:
> // That means the assembly was loaded successfully and Initialize method was called.

Maybe you could also iterate over the list of loaded assemblies to also assert that `OpenTelemetry.ClrProfiler.Managed.dll` has been loaded?

Reply:
I'm trying to think of an additional signal to assert that we've loaded everything correctly, in addition to no exceptions happening

Reply:
I actually couldn't come up with anything, if there is a problem with loading the assembly or calling the method the exception will be thrown, so I assume that is a valid thing to check against.

Reply:
I have added checking if the assembly is present in the list of the loaded assemblies.